db/schema.rb

```diff
@@ -19,8 +19,22 @@
     t.string "title"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "body"
+    t.string "author"
```

Comment:
Author should be the user's name. Don't store this here. Instead use the relationship to users to get their name.